personalrobotics/ada_feeding
# Taylorkf getting started

# Description

This PR is updating the README instructions and adding a requirements.txt file so that users without sudo access can get started without the assistance of someone with sudo access.

# Testing procedure
Create a new user account without sudo access and follow the instructions on the README. Ensure that the new user can run code on the robot and in simulation without sudo access. NOTE: I have currently only tested that a new user can run in simulation.

# Before opening a pull request

# Before Merging
